### ISSUE TITLE
Submit contribution

### ISSUE THREAD

**Issue by manwar** [manwar/theweeklychallenge#135]
Add Dancer2 route to enable user submit contributions either upload the script or type in the code.

**Comment by NoubissiViany** [comment]
@manwar can you please assign this task to me and give more details please

**Comment by soniyaprasad77** [comment]
I would love to work on this issue, please assign this to me.

**Comment by manwar** [comment]
![IMG_0116](https://github.com/manwar/theweeklychallenge/assets/681195/997afe83-0f9a-4715-a474-81b38921e7aa)

**Comment by NoubissiViany** [comment]
@manwar so this is a frontend task right???

**Comment by manwar** [comment]
@NoubissiViany yes it is, right now focus is to get prototype fully functional.

**Comment by NoubissiViany** [comment]
@manwar okay thanks
